config/default.js

```diff
@@ -22,46 +22,46 @@ module.exports = {
   //
   // Server
   //
-  server: {
-    port: 3000,
-    router: {
-      stripTrailingSlash: true
+  server : {
+    port   : 3000,
```

Comment:
Is this kind of formatting also a best practice? (same as `{ key : 'val' }` - spaces around `:`)